### PR TITLE
Fix #195, Don't log ssl.CertificateError

### LINKED ISSUE #195: Don't log ssl certificate errors
See https://github.com/python/cpython/pull/9169 for the upstream change

### PR DESCRIPTION
**Pull request by fantix**
Refs #195, refs https://github.com/python/cpython/pull/9169

### CODE REVIEW

File: uvloop/includes/stdlib.pxi [hunk 1]
```diff
@@ -126,6 +126,13 @@ cdef int ssl_SSL_ERROR_WANT_READ = ssl.SSL_ERROR_WANT_READ
 cdef int ssl_SSL_ERROR_WANT_WRITE = ssl.SSL_ERROR_WANT_WRITE
 cdef int ssl_SSL_ERROR_SYSCALL = ssl.SSL_ERROR_SYSCALL
 
+cdef FATAL_ERROR_IGNORE = (
```
Comment (1st1) [inline uvloop/includes/stdlib.pxi]: Can we rename this to FATAL_SSL_ERROR_IGNORE

Comment (fantix) [reply uvloop/includes/stdlib.pxi]: Makes sense! Will do, thx.

**Comment by 1st1** [comment]
What's the status on this?

**Comment by fantix** [comment]
There it is! 😃

**Pull request merged**